Lib/test/test_fcntl.py

```diff
@@ -34,7 +34,7 @@ def get_lockdata():
                                fcntl.F_WRLCK, 0)
     elif sys.platform.startswith('gnukfreebsd'):
         lockdata = struct.pack('qqihhi', 0, 0, 0, fcntl.F_WRLCK, 0, 0)
-    elif sys.platform in ['aix3', 'aix4', 'hp-uxB', 'unixware7']:
+    elif sys.platform in ['hp-uxB', 'unixware7']:
```

Comment:
Should not 'aix5', 'aix6', etc be added instead?

Reply:
replied by email, but to be sure - adding here as well.

-- No --. This has never been an issue with any AIX version 5, 6 or 7. This is probably something that got corrected as IBM added "Linux Affinity" back in 1999-2000.